vm/starnix/starnix.go

```diff
@@ -73,10 +75,17 @@ func ctor(env *vmimpl.Env) (vmimpl.Pool, error) {
 		return nil, fmt.Errorf("invalid config param count: %v, want [1, 128]", cfg.Count)
 	}
 
+	ffxDir, err := os.MkdirTemp("", "syz-ffx")
+	if err != nil {
+		return nil, fmt.Errorf("failed to make ffx isolation dir: %w", err)
+	}
+	log.Logf(0, "initialized vm pool with ffx dir: %v", ffxDir)
```

Comment:
Nit: how about putting this under an `if env.Debug`? (I do sometimes refer to this log string in order to run ffx log on the side, but it would be okay to have it only in debug mode.)

Reply:
Done!